.github/workflows/main.yml

```diff
@@ -302,7 +312,7 @@ jobs:
     container: alpine:3.16.0
     steps:
       - name: install tools
-        run: apk update && apk add build-base cmake git linux-headers perl tar python3 py3-pip
+        run: apk update && apk add build-base cmake git linux-headers perl tar python3 py3-pip py3-future
```

Comment:
future should come with pip, no need to install it.

Reply:
Ah, it's this step:
https://github.com/mavlink/mavlink/blob/master/.github/workflows/test_and_deploy.yml#L67-L68

Reply:
Sorry I'm confused, you're pointing to a workflow where future is installed with pip instead of the system package manager. Do you mean that installing with `apk` is a bad idea and it should be installed with `pip` instead? I really don't mind in the CI :thinking: